src/utils/pools.tsx

```diff
@@ -1,28 +1,10 @@
-import {
-  Account,
-  AccountInfo,
-  Connection,
-  PublicKey,
-  SystemProgram,
-  TransactionInstruction,
-} from "@solana/web3.js";
-import { sendTransaction, useConnection } from "./connection";
-import { useEffect, useMemo, useState } from "react";
-import { Token, MintLayout, AccountLayout } from "@solana/spl-token";
-import { notify } from "./notifications";
-import {
-  cache,
-  getCachedAccount,
-  useUserAccounts,
-  useCachedPool,
-  getMultipleAccounts,
-} from "./accounts";
-import {
-  programIds,
-  SWAP_HOST_FEE_ADDRESS,
-  SWAP_PROGRAM_OWNER_FEE_ADDRESS,
-  WRAPPED_SOL_MINT,
-} from "./ids";
+import { Account, AccountInfo, Connection, PublicKey, SystemProgram, TransactionInstruction } from '@solana/web3.js';
```

Comment:
This file got auto formatted by our new pretty standards, so Im going to mark with comments where to look for actual changes.